lesson4/backend28.js

```diff
@@ -3,9 +3,11 @@ const express= require('express')
 const app = express();
 app.listen(3618);
 app.use(express.static('public'));
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
 
-app.get('/send',(req,res)=>{
-  const firstdata = req.query.userNames;
-  const seconddata = req.query.userComments;
+app.post('/send',(req,res)=>{
+  const firstdata = req.body.name;
+  const seconddata = req.body.comment;
   fs.appendFile('/home/kinue/garageScript/lesson4/public/storing.txt', firstdata + ':' + seconddata + '\n')
```

Comment:
Use back ticks like this: `${firstData}: ${seconddata}` + '\n'

Reply:
this works as well, right?   

`${firstData} : ${secondData}\n`

Reply:
Yes.